admin/src/main/resources/io/buoyant/admin/js/utils.js

```diff
@@ -53,8 +53,8 @@ BytesToStringConverter.prototype.convert = function(b) {
  * Success Rate Utils
  */
 
-function SuccessRate(requests, success, failures) {
-  this.requests = requests;
+function SuccessRate(success, failures) {
+  this.requests = success + failures;
```

Comment:
now that requests is deterministic based on success+failure, the `requests` element of this data structure is redundant information. consider removing and allowing the caller to perform the addition?